layer.md

```diff
@@ -186,6 +186,38 @@ rootfs-c9d-v1.s1/
         my-app-tools
 ```
 
+### Symbolic Links
+
+When applying a changeset, (assessing the changeset, or creating the original comparison filesystem) symlinks that are either:
+
+- Already existing in a previous layer
+- Added in the current layer
+
+SHOULD NOT be followed. 
```

Comment:
let's move this sentence fragment into the line 191, like "symlinks SHOULD NOT be follow that are either:"